lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1310,7 +1310,8 @@ def test_view_certificate_link(self):
         )
         self.assertNotContains(resp, u"View Your Certificate")
         self.assertNotContains(resp, u"You can now view your certificate")
-        self.assertContains(resp, u"We're creating your certificate.")
+        self.assertIn("working on it...", resp.content)
+        self.assertIn("creating your certificate", resp.content)
```

Comment:
since `resp.content` is HTML, I think `assertContains` is perhaps more correct (although, since you don't have any HTML in your needle, perhaps it doesn't matter).